eng/common/scripts/Prepare-Release.ps1

```diff
@@ -64,6 +64,55 @@ function Get-ReleaseDay($baseDate)
   return $baseDate;
 }
 
+function Check-ApiReviewStatus($packageName, $language)
+{
+  # Get API view URL and API Key to check status
+  Login-AzAccount
+  $url = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIURL -AsPlainText
+  $apiKey = Get-AzKeyvaultSecret -VaultName "AzureSDKPrepRelease-KV" -Name APIKEY -AsPlainText
+
+  $lang = $language
+  # Update language name to match those in API cosmos DB. Cosmos SQL is case sensitive and handling this within the query makes it slow.
+  if($lang -eq 'javascript'){
```

Comment:
We have tried to avoid duplicating these language mappings in various places and usually have them defined in Language-settings.ps1. Do you have already have this mapping for your other ApiView work or did those not require the language?

Reply:
Language name is not required when we upload artifact to API view to create a review. Language name is identified based on the artifact type within APIView itself. But I totally understand your point and will make a helper in API view specific helper script module.